network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/listing_block.html

```diff
@@ -22,7 +33,16 @@ <h2 class="tw-h3-heading tw-mb-0 tw-pb-8 medium:tw-pb-16">{{ self.heading }}</h2
                            aria-label="{{ card.title }}">
                     {% endif %}
                     <div class="card-regular tw-h-full tw-flex tw-flex-col">
-                        <img src="{{ img.url }}" alt="{{ card.alt_text }}"/>
+                        <picture>
+                            {% image card.image fill-350x197 as img_small %}
+                            {% image card.image fill-700x394 as img_small_2x %}
+                            {% image card.image fill-510x287 as img_large %}
+                            {% image card.image fill-1020x574 as img_large_2x %}
+                            {# Larger screens use smaller image because multiple images are displayed per row, making each image smaller. #}
+                            <source media="(min-width: 768px)" srcset="{{ img_small.url }}, {{ img_small_2x.url }} 2x">
+                            <source srcset="{{ img_large.url }}, {{ img_large_2x.url }} 2x">
+                            <img src="{{ img_large.url }}" alt="{{ card.alt_text }}" width="{{ img_large.width }}" height="{{ img_large.height }}">
+                        </picture>
```

Comment:
I noticed that the image look washed out on small screens. This should fix it and use higher resolution on small screens as the image is effectively bigger.